src/bokeh/layouts.py

```diff
@@ -187,8 +187,40 @@ def layout(*args: UIElement, children: list[UIElement] | None = None, sizing_mod
     _children = _parse_children_arg(*args, children=children)
     return _create_grid(_children, sizing_mode, **kwargs)
 
+
+@overload
+def gridplot(
+    children: list[UIElement | None],
+    *,
+    sizing_mode: SizingModeType | None = None,
+    toolbar_location: LocationType | None = "above",
+    ncols: int,
+    width: int | None = None,
+    height: int | None = None,
+    toolbar_options: dict[ToolbarOptions, Any] | None = None,
+    merge_tools: bool = True,
+) -> GridPlot:
+    ...
+
+
+@overload
+def gridplot(
+    children: list[list[UIElement | None]],
+    *,
+    sizing_mode: SizingModeType | None = None,
+    toolbar_location: LocationType | None = "above",
+    ncols: None = None,
+    width: int | None = None,
+    height: int | None = None,
+    toolbar_options: dict[ToolbarOptions, Any] | None = None,
+    merge_tools: bool = True,
+) -> GridPlot:
+    ...
```

Comment:
@mattpap Codecov marks this as not coverd. Do you think this is fine or should we do something.

I found this comment https://github.com/nedbat/coveragepy/issues/970#issuecomment-612602180 where a file named `.coveragerc` is mentioned. In this file we can force coverage to ignore the lines with `@overload`.

```
[report]
exclude_lines = 
    pragma: not covered
    @overload
```

All other tests are green now.

Reply:
Good to know and let's add that.

Reply:
You can also add it to `pyproject.toml`,  `exclude_also` is an alternative where you don't need to add the default excludes lines. 

``` toml
[tool.coverage]
exclude_also = ["pragma: not covered", "@overload"]
```

Reply:
That's perhaps even better, keeping various tool configurations in one place.

Reply:
I successfully updated the `pyproject.toml` to ignore `@overload`. The never [`exclude_also`](https://coverage.readthedocs.io/en/latest/config.html#config-report-exclude-also) is preferred instead of `exclude_lines` and doesn't need `"pragma: not covered"` anymore.

In summery I adapted existing code in `pyproject.toml` and added

```
[tool.coverage.report]
exclude_also = ["@overload"]
```